speech/api/TranscribeContext/TranscribeContextClass.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START speech_adaptation]
+
+using System;
+using Google.Cloud.Speech.V1;
+
+namespace GoogleCloudSamples
+{
+    public class TranscribeContextClass
```

Comment:
nit: Consider renaming to TranscribeContextClasses (the file as well)

Reply:
due to overlapping method name, cannot assign that name. I will come up with differnet class name

Reply:
How about speechContextClasses